.travis.yml

```diff
@@ -1,6 +1,26 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-sudo: true
+sudo: required
+dist: trusty
+before_install:
+    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
+    - sudo apt-get update -qq
+    - sudo apt-get install libgcrypt20 liblzma5 libselinux1
+    - wget "http://archive.ubuntu.com/ubuntu/ubuntu/pool/main/s/systemd/libsystemd0_219-7ubuntu3_amd64.deb"
+    - wget "http://archive.ubuntu.com/ubuntu/ubuntu/pool/main/s/systemd/libsystemd-dev_219-7ubuntu3_amd64.deb"
+    - sudo dpkg -i libsystemd0_219-7ubuntu3_amd64.deb libsystemd-dev_219-7ubuntu3_amd64.deb
+    - git clone https://github.com/xapi-project/xs-opam && cd xs-opam && sh into_repo.sh && cd -
```

Comment:
A minor thing. In one of the other "ports", to avoid issues with `ocveralls` I had to change this line as
```
git clone https://github.com/xapi-project/xs-opam /tmp/xs-opam && cd /tmp/xs-opam && sh into_repo.sh && cd -
```
The REMOTE paths will become `/tmp/xs-opam` instead of `$TRAVIS_BUILD_DIR/xs-opam`.

In this case, without coverage, it does not make any difference, so I leave it up to you if you want to change it or not. If not, I am happy to merge it now that the test is passed.